rdkit/SimDivFilters/SimilarityPickers.py

```diff
@@ -110,8 +112,10 @@ def __init__(self, numToPick=10, probeFps=None, dataSet=None,
     self.simMetric = simMetric
     self._picks = None
 
-  def MakePicks(self, force=0):
+  def MakePicks(self, force=False, silent=True):
     if self._picks is not None and not force:
+      if not silent:
+        print('Use force to re-make picks')
```

Comment:
This is actually worse here. I've added this to allow testing the force. No one would really want to see this message.